web/gds-user-ui/src/components/RegistrationForm/InvalidFormPrompt.tsx

```diff
@@ -25,6 +25,7 @@ function InvalidFormPrompt({
   handleContinueClick,
   isNextStep
 }: InvalidFormPromptProps) {
+  console.log("isNextStep: " + isNextStep);
```

Comment:
I couldn't get the error we saw in staging to appear locally. When checking `isNextStep` the correct Boolean value was returned, but I think keeping it for now could help if this change doesn't fix the error.